### DISCUSSION TITLE
Welcome to PandaCrazy-Max Discussions!

### DISCUSSION THREAD

**Discussion by JohnnyRS** [JohnnyRS/PandaCrazy-Max#32] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by leemthai** [comment]
I'm lee and have been using Panda Crazy for the last 3 years so looking forward to testing out Max ASAP. I'll be sure to have  some questions and feedback once I've had a chance to start using it :)

**Comment by seasons1981** [comment]
I have a few recommendations, I hope they make sense to you (lol): 

** block button on both search and max.  <sometimes you get hits that come in that you do not qualify for, but with the way that the triggers work, they come in many times throughout the day  (same GID) it would be beneficial for us to have a quick "block GID/RID" button so they stop coming in> -- I know we have the option to put them into the search area ourselves, but when you have a full queue, sometimes it is hard to remember the name of the hit or requester.

** have a button to collect ONLY ONCE - ALL DAY LONG.  As in, even if I clean out my max of all the things no longer collecting, it won't pull in a hit I have previously done that day, unless I turn it on myself in max. This would save my panda a lot of time trying to collect the same thing over and over and over again just because I cleaned up my max area.  I get so many of the same thing over and over again because a new hit comes in on search.  (I understand if it has a new GID, but with the same GID showing up multiple times, it catches it every time.)

**on SEARCH side, under "Custom Triggered Hits" it would be amazing to have a "search" feature specific to the hits that have come in, so I can find a specific hit after removing it from MAX.

Anyway, I appreciate all the work you put in for us. So thank you very much for taking our feedback and working your magic to help us work smarter, not harder.

**Reply by seasons1981** [reply]
forgot to add... it would be amazing to also start at $0.00 for triggers. It is set to $0.01 but some great hits have $0 and a good bonus.  We miss these because of the penny start.

**Reply by seasons1981** [reply]
I keep remembering things to add... so I will keep replying to mine with them.

it would be great to see the "hourly rate" for TV/TO and maybe even have the triggers accept only things with your choice of rating.  I know you have the pricing point, but to have both would be great!

**Reply by JohnnyRS** [reply]
Just a few quick things on some stuff:
Qualified hits are done by filters on mturk search page which the script uses. Be sure you have it only showing qualified hits. I can place checks for qualification also but that just adds more things to check which can slow things down.

There is a little bug with once that has been fixed but I haven't pushed it through chrome just yet. I can see where a once daily thing could be nice. Probably could work with a temporary variable as long as the browser doesn't restart that day it should do what you would want.

A search feature would be nice for the search triggered hits tab. Also I want to bring in more history stats which I will be working on next month because every hit that it sees is being saved but users can't get to it right now.

I could add a way to go down to $0 for users who want that just need to make sure it's not recommended for new users to mturk.

I have thought about TV/TO ratings but that gets complicated. To get the ratings it will take a second or more which defeats the purpose of trying to accept HITs as fast as possible. Only other way is to save the ratings in a database but then I need permission from TV/TO because that is their data they are creating which at this point I don't really want to deal with. But it is something I am thinking about in the future.

**Reply by seasons1981** [reply]
I understand the qualification part, but some requesters don't choose the qualifications.  It just comes in and when you go to the hit you realize you need to be a 25 year old male, but in all actuality you are a 40 year old female. (lol) that is on the requester to place in the qualifications, but sometimes they don't and don't explain it until you open their link.  Which brings me to what I was trying to say.  The blocking of that specific hit would be fantastic.

Otherwise, I completely understand the rest. 

Thanks for your quick response.  If I think of anything else, I will post again.

**Comment by jfoz37** [comment]
Love the extension and really appreciate your hard work. Have a small request I hope you'll consider next time you update.

I prefer using Hit Finder from Mturk Suite to search for hits. The hit logging feature and all of the additional info (like the requester approval rates) really comes in handy. 

Unfortunately, there is no easy way to send the hits from Hit Finder over to Panda Crazy. The Hit Cather part of Mturk Suite works poorly for me, if at all.  Panda Crazy is far more stable and vastly superior in my opinion. It is just a hassle to have to manually enter information for each Hit.

Mturk Suite has a context menu option that will allow you to send any link over to Hit Catcher. It would be a tremendous help if Panda Crazy could provide the same feature.

**Comment by jfoz37** [comment]
Love the extension and really appreciate your hard work. Have a small request I hope you'll consider next time you update.

I prefer using Hit Finder from Mturk Suite to search for hits. The hit logging feature and all of the additional info (like the requester approval rates) really comes in handy. 

Unfortunately, there is no easy way to send the hits from Hit Finder over to Panda Crazy. The Hit Cather part of Mturk Suite works poorly for me, if at all.  Panda Crazy is far more stable and vastly superior in my opinion. It is just a hassle to have to manually enter information for each Hit.

Mturk Suite has a context menu option that will allow you to send any link over to Hit Catcher. It would be a tremendous help if Panda Crazy could provide the same feature.

**Reply by JohnnyRS** [reply]
Thanks for requesting this feature and suggested using a context menu. It's been a long time but I finally looked at this and next version should have this added to the context menu. I'm working on it now and seems to work out nicely from Mturk Suite. Will work on any page with Mturk links too. Also I might add a way to create custom triggers by any selection.

**Reply by Iconology** [reply]
Hey, Johnny, first of all I appreciate all your work on this project! Is it possible to request a feature, simply exclude any requester with an approval rating below a specific value from showing up in SearchCrazy? Just a quick example, if a requester has an approval rating below 95% it will be excluded from appearing in the custom triggered hits tab. I tried to implement it but I'm quite new to programming so its beyond my skill-set.

**Comment by jfoz37** [comment]
That is great news! I really appreciate your hard work.
________________________________

**Comment by mikec7000** [comment]
Do any of your guys know how to fix this error. It keep getting stuck in the "Loading Data" data phase infinitely no matter if you turn off and on the addon and the only way to fix it is I always have to remove the addon each time and install it again. I tried using the "Reset" and "Wipe Data" buttons and no luck.

Loading Data
×
Please Wait. Loading up all data for you.

**Comment by djalski** [comment]
Hello, Can anyone help me out. What am i doing wrong. I keep getting this error when i click on search.
There are no search triggers enabled to search for so searching cancelled.
I have created a search but maybe i did it wrong. 
I read the wiki but that doesn't really explain it or i'm not understanding it.

Thank you

**Comment by gopik25** [comment]
Sub : Goham fetch is slow 
Hi,
When click the Goham fetch count speed is slow and sometimes fetch is stuck
so that's why Hits are not catching , the catching possible is very low,
please check and solve the problem.
Thank you
